### DISCUSSION TITLE
Provide shaders in the new format omwfx?

### DISCUSSION THREAD

**Discussion by franzmondlichtmann** [zesterer/openmw-shaders#7] [Ideas]
Will you eventually implement your Shaders in this new Format called omwfx?
It would be nice if I could edit the shaders while the game is running.
And it would make swapping shaders much easier and faster.

Quoted from https://openmw.org/en/:
"OpenMW now provides a new file format called omwfx where the post-processing shaders are stored. This means shader developers have an easy way to create and tweak shaders that end users can easily install and manage for their games. Each shader can be turned on and off in one click and can even be fine-tuned through the super-handy in-game interface. The shaders can also be "hot reloaded", i.e, you can edit the shaders while the game is running. This is, of course, very useful for developers so that they can see the result of what they just wrote in real time without restarting the game."

**Comment by zesterer** [comment]
`.omwfx` is strictly for post-processing shaders, which these are not. For the foreseeable future, OpenMW doesn't really plan to support core shaders as mods. However, my [cloud and mist shader](https://github.com/zesterer/openmw-volumetric-clouds), which is a post-processing shader, is already in `.omwfx` format.
